packages/front-end/components/Experiment/TabbedPage/index.tsx

```diff
@@ -281,116 +283,120 @@ export default function TabbedPage({
-            mutate={mutate}
-            disableEditing={viewingOldPhase}
-            linkedFeatures={linkedFeatures}
-            visualChangesets={visualChangesets}
-            editTargeting={editTargeting}
-            verifiedConnections={verifiedConnections}
-            checklistItemsRemaining={checklistItemsRemaining}
-            setChecklistItemsRemaining={setChecklistItemsRemaining}
-            envs={envs}
-          />
-          <Implementation
-            experiment={experiment}
-            mutate={mutate}
-            safeToEdit={safeToEdit}
-            editVariations={editVariations}
-            setFeatureModal={setFeatureModal}
-            setVisualEditorModal={setVisualEditorModal}
-            setUrlRedirectModal={setUrlRedirectModal}
-            visualChangesets={visualChangesets}
-            urlRedirects={urlRedirects}
-            editTargeting={editTargeting}
-            linkedFeatures={linkedFeatures}
-            envs={envs}
-          />
-          {experiment.status !== "draft" && (
-            <div className="mt-3 mb-2 text-center d-print-none">
-              <Button
-                onClick={() => setTabAndScroll("results")}
-                size="lg"
-                icon={<FaChartBar />}
-              >
-                View Results
-              </Button>
+          <div className="mt-3">
+            <CustomMarkdown page={"experiment"} variables={variables} />
+          </div>
+
+          {experiment.status === "stopped" && (
+            <div className="pt-3">
+              <StoppedExperimentBanner
+                experiment={experiment}
+                linkedFeatures={linkedFeatures}
+                mutate={mutate}
+                editResult={editResult || undefined}
+              />
             </div>
           )}
-        </div>
-        {isBandit ? (
+          {viewingOldPhase &&
+            ((!isBandit && tab === "results") ||
+              (isBandit && tab === "explore")) && (
+              <div className="alert alert-warning mt-3">
+                <div>
+                  You are viewing the results of a previous experiment phase.{" "}
+                  <a
+                    role="button"
+                    onClick={(e) => {
+                      e.preventDefault();
+                      setPhase(experiment.phases.length - 1);
+                    }}
+                  >
+                    Switch to the latest phase
+                  </a>
+                </div>
+                <div className="mt-1">
+                  <strong>Phase settings:</strong>{" "}
+                  {phaseSummary(experiment?.phases?.[phase])}
+                </div>
+              </div>
+            )}
           <div
-            className={
-              // todo: standardize explore & results tabs across experiment types
-              isBandit && tab === "results" ? "d-block" : "d-none d-print-block"
-            }
+            className={clsx(
+              "pt-3 where",
```

Comment:
Is this intentional?  I don't see any `where` class name in our styles.

Reply:
ah, debugging I forgot about... good catch